tests/testthat/test-paren_body_linter.R

```diff
@@ -43,4 +42,11 @@ testthat::test_that("paren_body_linter returns correct lints", {
 
   # multiple lints on the same line
   expect_lint("function()if(TRUE)while(TRUE)test", list(msg, msg, msg), linter)
+
+  # No space after the closing parenthesis of an anonymous function prompts a lint
+  testthat::skip_if(
+    compareVersion(paste(R.version$major, R.version$minor, sep = "."), "4.1.0") < 0,
```

Comment:
You can use `getRversion() >= "4.1"` since `package_version` implements a comparison.

Reply:
```suggestion
    getRversion() < "4.1",
```